files/emailservice/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Tim Yelland"
+github = "https://github.com/mboosh/emailservice"
+homepage = "https://github.com/mboosh/emailservice"
+description = ""
+mainfile = "src/main/webapp/js/app.js"
```

Comment:
Thank you megawac for all your help, appreciate it as a first time user of jdelivr.